src/main/java/net/sf/jabref/exporter/FieldFormatterCleanups.java

```diff
@@ -85,7 +85,7 @@ public int hashCode() {
         return Objects.hash(actions, enabled);
     }
 
-    private static List<FieldFormatterCleanup> parse(String formatterString) {
+    public static List<FieldFormatterCleanup> parse(String formatterString) {
```

Comment:
Yes, it should work, but there is a boolean parameter required which does not have any effect. The constructor call looks a bit puzzling. What should I prefer?